.github/workflows/dotnet-core-build-and-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
+   
+    - name: Install dependencies for fappdeploy
+      run: |
+           cd ./fappdeploy/
+           dotnet restore
+           
+    - name: Build fappdeploy
```

Comment:
I kept it separate on purpose so that if we make changes to specific files and there is an issue with that we'll know from the build itself. I think we should keep them separate only.

Reply:
this way it looks cleaner to me